ega-data-api-dataedge/src/main/resources/application.yml

```diff
@@ -1,15 +1,29 @@
-spring.profiles: no-oss
+spring.profiles.active: no-oss,logger-log,add-user-ip-headers
```

Comment:
I think that what you want to do is

spring.profiles: no-oss
spring.profiles.include:logger-log,add-user-ip-headers
that will include the other profiles if the no-oss is set

What you were doing was always loading that 3 profiles for everyone I think.

Reply:
ok, will change